src/lib/project.ts

```diff
@@ -58,7 +58,7 @@ export class Project {
                     .filter((x): x is ProjectTemplate => x !== undefined) ?? []
 
             for (const c of children) {
-                if (found[c.type] === undefined) {
+                if (!(c.type in found)) {
```

Comment:
this doesn't work for `{ [k: string]: string | undefined }` types, but I guess it works here because it's parsed JSON?